Tutorial updates (#3524)
dotansimha/graphql-yoga: website/src/pages/tutorial/basic/08-graph-relations.mdx

  id: ID!
  description: String!
  url: String!
  comments: [Comment!]
}
```

On the Prisma schema, we chose to make comments mandatory because of their best practices. So in

> case there are no comments, it will return an empty string.

Reviewer: On the Prisma API level, the method call for loading will always return an array. It does not distinguish between an empty array and an array with comments.

> On the Prisma schema, we chose to make comments mandatory because of their best practices. So in
> case there are no comments, it will return an empty string.

Reviewer: I an pretty sure it will not return an empty string.

```suggestion
On the Prisma API level, the method call for loading will always return an array. It does not distinguish between an empty array and an array with comments.
```

> On the GraphQL schema, we can choose to follow the same practice, or to make the comments array
> nullable.

Reviewer: ```suggestion
On the GraphQL schema, we could follow the same practice, and make the comments array
non-nullable (`[Comment!]!`).
```

> In our example, we've choosen the nullable option, as it single to the consumer of our API that they

Reviewer: ```suggestion
In our example, we've choosen the nullable option, as it indicates the API the
```
